shared_model/validators/protobuf/proto_block_validator.cpp

```diff
@@ -18,7 +24,26 @@ namespace shared_model {
           == iroha::protocol::Block::BLOCK_VERSION_NOT_SET) {
         reason.second.emplace_back("Block version is not set");
         answer.addReason(std::move(reason));
-        return answer;
+      }
```

Comment:
Seems we have to return the `answer` here if the block has no `block_version`, instead of examining non-existent data.